src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthorizationBackend.java

```diff
@@ -962,10 +964,10 @@ public void fillRoles(final User user, final AuthCredentials optionalAuthCreds)
                 for (final LdapName roleLdapName : nestedReturn) {
                     final String role = getRoleFromEntry(connection, roleLdapName, roleName);
 
-                    if (!Strings.isNullOrEmpty(role)) {
-                        user.addRole(role);
+                    if (excludeRolesMatcher.test(role)) {
+                        log.warn("Role was excluded or empty, attribute: '{}' for entry: {}", roleName, roleLdapName);
```

Comment:
Seems like an debug level message not a warning.  Since this is a feature to exclude these values, only when debugging/tracing would this seems to make sense to be verbose about

Reply:
I agree, I will change this to debug

Reply:
Changed to debug